build/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<title>Apache Arrow Cookbook</title>
+		<style>
+			body {
+				color: rgb(51, 51, 51);
+				font-family: sans-serif;
+				line-height: 1.65;
+				padding: 25px;
+				max-width: 900px;
+				margin-left: auto;
+				margin-right: auto;
+			}
+
+			a {
+				color: rgb(0, 91, 129);
+			}
+
+			#logo {
+				width: 50%;
+				margin-left: auto;
+				margin-right: auto;
+			}
+
+			#logo > img {
+				width: 100%;
+			}
+		</style>
+	</head>
+	<body>
+		<div id="logo"><img src="arrow.png"/></div>
+		<h1>Apache Arrow Cookbook</h1>
+		<p>The cookbook is a collection of Apache Arrow recipes for
+		   the languages and platform supported by Arrow.<br/>
+		   Most recipes will be common to all patforms, 
+		   but some are specific to the language and environment in use.
```

Comment:
Maybe put a pointer here that this is not documentation and to find documentation/reference on the libraries to go to arrow.apache.org/docs?

Reply:
Done in 8e0c3e0